packages/discord.js/src/structures/AutoModerationActionExecution.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+/**
+ * Represents the structure of an executed action when an {@link AutoModerationRule} is triggered.
+ */
+class AutoModerationActionExecution {
+  constructor(data, guild) {
+    /**
+     * The guild id where this action was executed from.
+     * @type {Snowflake}
+     */
+    this.guildId = data.guild_id;
```

Comment:
We have a `AutoModerationActionExecution#guild`, can't we make this property a getter of `this.guild.id`? Or what is the reasoning behind the redundancy?